web/Views/Property/ViewProperty.cshtml

```diff
@@ -1,132 +1,338 @@
-﻿@using Microsoft.AspNetCore.Mvc.TagHelpers
-@model PropertyViewModel
+﻿@model PropertyDetailsViewModel

```

Comment:
This file is very very long, it may benefit from being broken into some partials to just separate out some of the complexity and noise :)